src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,83 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * Class create to print items.
+ *
+ * @author Galvin Leow Wen Yuan
+ * @author A0200204J
+ * @version v1.0
+ */
+public class Ui {
+
+    /**
+     * Print out welcome message with the menu.
```

Comment:
I am just being nit-picky here but you can consider changing it to "Prints" instead of "Print" as described in the coding standard. I noticed this in several other places too